src/alliance_disease_association_ingest/transform.py

```diff
@@ -1,13 +1,24 @@
 import uuid
-
+import pandas as pd
 from biolink_model.datamodel.pydanticmodel_v2 import (
     GeneToDiseaseAssociation,
     GenotypeToDiseaseAssociation,
-    VariantToDiseaseAssociation, KnowledgeLevelEnum, AgentTypeEnum,
+    VariantToDiseaseAssociation,
+    KnowledgeLevelEnum,
+    AgentTypeEnum
 )
 from koza.cli_utils import get_koza_app
 
-#  TODO: look at row["source"] to update this map
+# Load the CSV file
+mapping_file = ".tsv" # MONDO-DOID mapping file
+mapping_df = pd.read_csv(mapping_file, delimiter='\t')
+
+# Create a mapping dictionary from DOID to MONDO
+doid_to_mondo_mapping = dict(zip(mapping_df['object_id'], mapping_df['subject_id']))
+
```

Comment:
We should probably change this up to to use the Koza built in SSSOM apply config, maybe with some points to doc/examples from @glass-ships